.github/workflows/check-pull-request.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v4.0.1
         with:
-          node-version: 18
```

Comment:
node 18 is still a major available version. We might want to test it

Reply:
I know it's still in the matrix, i changed only the version of node used to fetch node_modules

Reply:
quickly discussed with @Codelax ; we suggest to change in the matrix 19 to 20, and lgtm :)